internal/server/v1alpha1/handlers.go

```diff
@@ -73,8 +74,19 @@ func webhookService(s *Server, spec *config.WebhookSpec, r *http.Request) error
 		}
 	}
 
-	// TODO Do the webhook storage
-	s.logger.Warn().Msg("Storage not implemented yet")
+	if r.Body != nil {
```

Comment:
Use guard clause instead at the start of function
```go
if (r.Body == nil) {
  return errors.New("Request don't have body")
}
```

Reply:
I have problem when i run test, the body send is nil at all time, but for send non nil body need to send `body io.Reader`

`func NewRequest(method, url string, body io.Reader) (*Request, error) {`

But in the test the request is created like that:
`	req, err := http.NewRequest("POST", "/v1alpha1/test", nil)`

Reply:
You can send an hardcoded body with. `strings.NewReader("{}")`. 

The NewRequest is in a helper, all extra params to the helper to pass an `*io.Reader` and let each tests the possibility of configure it.